package.json

```diff
@@ -48,6 +48,7 @@
     "@webcomponents/custom-elements": "^1.2.4",
     "babel-loader": "^8.0.6",
     "babel-plugin-transform-react-remove-prop-types": "^0.4.24",
+    "blurhash": "^1.1.3",
```

Comment:
We could try https://github.com/fpapado/blurhash-rust-wasm if performance turns out to be an issue.